app/src/test/kotlin/fr/free/nrw/commons/upload/FileProcessorTest.kt

```diff
@@ -35,4 +39,50 @@ class FileProcessorTest {
     fun processFileCoordinates() {
 
     }
+
+    /**
+     * Test method to verify redaction Exif metadata
+     */
+    @Test
+    fun redactExifTags() {
+
+        val filePathRef: String? = "src/test/data/exif_redact_sample.jpg"
+        val filePathTmp: String? = "" + System.getProperty("java.io.tmpdir") + "exif_redact_sample_tmp.jpg"
+
+        val inStream = FileInputStream(filePathRef)
+        val outStream = FileOutputStream(filePathTmp)
```

Comment:
See the details of the test results, it says:
```
fr.free.nrw.commons.upload.FileProcessorTest > redactExifTags FAILED
    java.io.FileNotFoundException at FileProcessorTest.kt:53
```

Reply:
I see. I removed it. Help needed. Sorry.

Reply:
Sure @VitalyVPinchuk , let me investigate. Thanks for your fast responses.

Reply:
Hi @neslihanturan 
Any luck investigating the problem?

Reply:
Hi @VitalyVPinchuk sorry for the delay. Currently I have another task, I am planning to finalize this one at May 18. Thanks for keeping the pr up to date.

Reply:
Hi @VitalyVPinchuk , I am still busy with my current refactor task. Didn't forget about this one though:)